### ISSUE TITLE
Failure to start

### ISSUE THREAD

**Issue by aleixpol** [IsmaelMartinez/teams-for-linux#1197]
**Describe the bug**
At startup, I get an empty white screen saying with title: "Waiting for networks".

**To Reproduce**
1. Install from flathub (flatpak)
2. Start the application

**Expected behavior**
I assume a login screen?

**Screenshots**
![teams](https://github.com/IsmaelMartinez/teams-for-linux/assets/1282478/5278d55b-9adc-4cb8-b62a-37394ddc7d99)

**Desktop (please complete the following information):**
 - OS: Linux
 - Installation package: Flathub
 - Version: 1.4.25

**Debug**

```bash
$ flatpak run com.github.IsmaelMartinez.teams_for_linux --webDebug  --appLogLevels=error,info,warn,debug
[DEBUG] configPath:,/lhome/apolgon/.var/app/com.github.IsmaelMartinez.teams_for_linux/config/teams-for-linux
[WARN] No config file found, using default values
[DEBUG] configFile:,[object Object]
[INFO] Running under Wayland, switching to PipeWire...
LaunchProcess: failed to execvp:
xdg-settings
[INFO] App already running
Gtk-Message: 10:10:45.973: Failed to load module "xapp-gtk3-module"
[2:0415/101046.667005:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0415/101046.667269:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0415/101046.684027:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
```

**Comment by jijojosephk** [comment]
This `[INFO] App already running` means already an instance running. It's not related to the problem you're facing. But can you terminate the running app and start with `flatpak run com.github.IsmaelMartinez.teams_for_linux --webDebug  --appLogLevels=error,info,warn,debug` ?

**Comment by jijojosephk** [comment]
You can  join the matrix chat room: https://matrix.to/#/#teams-for-linux_community:gitter.im

**Comment by jijojosephk** [comment]
Also let us know if you're using proxy server.

**Comment by aleixpol** [comment]
```
[2:0415/114154.278314:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[DEBUG] configPath:,/lhome/apolgon/.var/app/com.github.IsmaelMartinez.teams_for_linux/config/teams-for-linux
[WARN] No config file found, using default values
[DEBUG] configFile:,[object Object]
[INFO] Running under Wayland, switching to PipeWire...
LaunchProcess: failed to execvp:
xdg-settings
Gtk-Message: 11:41:54.474: Failed to load module "xapp-gtk3-module"
[2:0415/114154.512810:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0415/114154.512843:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[DEBUG] Fetching custom background configuration from 'http://localhost/config.json'
[DEBUG] Custom background service url is 'http://localhost'
[DEBUG] processArgs:,/app/teams-for-linux/teams-for-linux,--webDebug,--appLogLevels=error,info,warn,debug
[2:0415/114154.575829:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0415/114154.575855:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0415/114154.575873:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[DEBUG] Testing network using net.request() for https://teams.microsoft.com/
[DEBUG] Spellchecker is disabled!
[2:0415/114154.697130:ERROR:browser_main_loop.cc(278)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[2:0415/114154.697253:ERROR:browser_main_loop.cc(278)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[2:0415/114154.697308:ERROR:browser_main_loop.cc(278)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[2:0415/114154.697375:ERROR:browser_main_loop.cc(278)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[2:0415/114154.698170:ERROR:browser_main_loop.cc(278)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[ERROR] Failed to fetch custom background remote configuration. Server returned error code '404'
[51:0415/114200.099141:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
```

Yes, this is behind certain proxies.

I've joined the matrix channel

**Comment by lxndrdnxl** [comment]
Hi, it seems like I am having a similar problem on Linux Mint 21.2
`
09:25:49 alex@alex-workmachine ~ → flatpak run com.github.IsmaelMartinez.teams_for_linux --webDebug  --appLogLevels=error,info,warn,debug
[2:0430/092551.721511:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[DEBUG] configPath:,/home/alex/.var/app/com.github.IsmaelMartinez.teams_for_linux/config/teams-for-linux
[WARN] No config file found, using default values
[DEBUG] configFile:,[object Object]
LaunchProcess: failed to execvp:
xdg-settings
Gtk-Message: 09:25:51.972: Failed to load module "xapp-gtk3-module"
[2:0430/092552.040602:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0430/092552.040642:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[DEBUG] Fetching custom background configuration from 'http://localhost/config.json'
[DEBUG] Custom background service url is 'http://localhost'
[DEBUG] processArgs:,/app/teams-for-linux/teams-for-linux,--webDebug,--appLogLevels=error,info,warn,debug
[2:0430/092552.198274:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0430/092552.198333:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0430/092552.198384:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[DEBUG] Testing network using net.request() for https://teams.microsoft.com/
[DEBUG] Spellchecker is disabled!
`

Are there any updates on that issue?

**Comment by IsmaelMartinez** [comment]
Hi @lxndrdnxl , we didn't get very far with this issue and @aleixpol as he wasn't able to run the app using anything other than flatpack. I would suggest you open another issue using the template so we can see if we can give you a hand to sort if and/or find why is not working. Closing this issue to avoid others reusing it.

**Comment by IsmaelMartinez** [comment]
This might be now fixed in [pre-release 1.9.1](https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.9.1) using the config option `disableGpu` that was missing an extra electron command. See https://github.com/IsmaelMartinez/teams-for-linux/issues/1369 for more info

**Comment by markus-haehnel** [comment]
I have the same issue. After deleting the user config (i. e. `rm -r ~/snap/teams-for-linux`) and a fresh login, it worked again.
